apps/web/src/routes/NonAuthLayoutRouter/ManageNotifications/ManageNotifications.js

```diff
@@ -55,115 +55,120 @@ export default function ManageNotifications (props) {
-                <SelectContent>
-                  <SelectItem value='daily'>{t('Daily')}</SelectItem>
-                  <SelectItem value='weekly'>{t('Weekly')}</SelectItem>
-                  <SelectItem value='never'>{t('Never')}</SelectItem>
-                </SelectContent>
-              </Select>
-            </div>
-            <div className={styles.settingWrapper}>
-              <label className={styles.settingExplanation}>{t('Send notifications for each new post in your group?')}</label>
-              <Select
-                value={unsubscribeAll ? 'none' : postNotifications}
-                onValueChange={value => updateSetting('postNotifications')(value)}
-                disabled={unsubscribeAll}
-              >
-                <SelectTrigger className='inline-flex w-auto'>
-                  <SelectValue />
-                </SelectTrigger>
-                <SelectContent>
-                  <SelectItem value='none'>{t('No Posts')}</SelectItem>
-                  <SelectItem value='important'>{t('Announcements & Mentions only')}</SelectItem>
-                  <SelectItem value='all'>{t('Every Post')}</SelectItem>
-                </SelectContent>
-              </Select>
-            </div>
-            <div className={styles.settingWrapper}>
-              <label className={styles.settingExplanation}>{t('Send notifications about comments on posts you are following via')}</label>
-              <Select
-                value={unsubscribeAll ? 'none' : commentNotifications}
-                onValueChange={value => updateSetting('commentNotifications')(value)}
-                disabled={unsubscribeAll}
-              >
-                <SelectTrigger className='inline-flex w-auto'>
-                  <SelectValue />
-                </SelectTrigger>
-                <SelectContent>
-                  {notificationOptions.map(option => (
-                    <SelectItem key={option.id} value={option.id}>{option.label}</SelectItem>
-                  ))}
-                </SelectContent>
-              </Select>
-            </div>
-            <div className={styles.settingWrapper}>
-              <label className={styles.settingExplanation}>{t('Send notifications for direct messages via')}</label>
-              <Select
-                value={unsubscribeAll ? 'none' : dmNotifications}
-                onValueChange={value => updateSetting('dmNotifications')(value)}
-                disabled={unsubscribeAll}
-              >
-                <SelectTrigger className='inline-flex w-auto'>
-                  <SelectValue />
-                </SelectTrigger>
-                <SelectContent>
-                  {notificationOptions.map(option => (
-                    <SelectItem key={option.id} value={option.id}>{option.label}</SelectItem>
-                  ))}
-                </SelectContent>
-              </Select>
-            </div>
+    <>
+      <Helmet>
+        <title>{t('Manage Notifications')}</title>
```

Comment:
I think we always want Hylo to be in the page title somewhere, the way we have been doing it is Page Name | Hylo